app/src/main/java/org/schabi/newpipe/player/BackgroundPlayer.java

```diff
@@ -356,12 +403,11 @@ public void onUpdateProgress(int currentProgress, int duration, int bufferPercen
             updateProgress(currentProgress, duration, bufferPercent);
 
             if (!shouldUpdateOnProgress) return;
-            if (timesNotificationUpdated > NOTIFICATION_UPDATES_BEFORE_RESET) {
-                resetNotification();
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O /*Oreo*/) updateNotificationThumbnail();
-            }
+            if (timesNotificationUpdated > NOTIFICATION_UPDATES_BEFORE_RESET) {resetNotification();
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O /*Oreo*/)
+                updateNotificationThumbnail();}
```

Comment:
This is really messy, why was it refactored this way? ;-)
I think it should be transformed into
```
            if (timesNotificationUpdated > NOTIFICATION_UPDATES_BEFORE_RESET) {
                resetNotification();
                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O /*Oreo*/) {
                    updateNotificationThumbnail();
                }
            }
```

Reply:
Whoops, I guess this has happened after auto merging. Fixed.